libcentrifugo/engine.go

```diff
@@ -6,16 +6,18 @@ type historyOpts struct {
 	Limit int
 }
 
-type addHistoryOpts struct {
-	// Size is maximum size of channel history that engine must maintain.
-	Size int
-	// Lifetime is maximum amount of seconds history messages should exist
+type publishOpts struct {
+	// Message is Message being published
+	Message Message
```

Comment:
Having actual message payload in the "options" configuration seems odd to me. As it's also separate param to publish as expected. Is this left over from an aborted refactor?